pkg/controller/cluster_controller.go

```diff
@@ -75,6 +78,7 @@ type ClusterReconciler struct {
 // +kubebuilder:rbac:groups=batch,resources=jobs/finalizers,verbs=get;list;watch
 // +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=mutatingwebhookconfigurations,verbs=get;update;patch;
 // +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingwebhookconfigurations,verbs=get;update;patch;
+// +kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,verbs=use
```

Comment:
This is needed because the operator SA can't grant permissions it doesn't have. Since we grant `use security.openshift.io/securitycontextconstraints` to the database role, we need it here as well.